.gitignore

```diff
@@ -35,7 +35,7 @@ hspec-results.md
 /docs/static/haddock/
 
 # demo
-devnet
+/devnet
```

Comment:
fyi, this was an important fix because this banned a "devnet" folder _anywhere_; not just at the root.